src/components/FishTankSwitch.vue

```diff
@@ -116,18 +116,18 @@ export default Vue.extend({
 })
 </script>
 
-<style lang="scss">
+<style module lang="scss">
   @import '../styles/mixins';
   @import "../../node_modules/@fishtank/colors/dist/index";
   @import "../../node_modules/@fishtank/type/dist/index";
 
 
 
-  body.user-is-tabbing input:focus + .input-checkbox-wrap {
+  body.user-is-tabbing input:focus + .inputCheckboxWrap {
```

Comment:
.user-is-tabbing is a global utility class name use for accessibility. Lets discuss how to make this a little more functional with CSS Modules.